translate/cloud-client/translate_v3_create_get_delete_glossary_test.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import translate_v3_create_glossary
+import translate_v3_delete_glossary
+import translate_v3_get_glossary
+import uuid
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+GLOSSARY_INPUT_URI = "gs://cloud-samples-data/translation/glossary_ja.csv"
+
+
+@pytest.fixture(scope="session")
+def glossary():
```

Comment:
These should be separate tests for each snippet. Each snippet should verify that it's functionality is complete.

Reply:
Okay, I will break it down into 3 separate tests